currency-conversion-service/build.gradle

```diff
@@ -23,13 +23,17 @@ repositories {
 }
 
 ext {
-    set('springCloudVersion', "Hoxton.SR1")
+    set('springCloudVersion', "Greenwich.SR3")
 }
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.cloud:spring-cloud-starter-config'
+    implementation ('org.springframework.cloud:spring-cloud-starter-openfeign', {
```

Comment:
Open feign has transitive dependency on ribbon. This has to be excluded to make the actual ribbon dependency to work. Otherwise It will fail to start.